code/modules/projectiles/magazines/shotguns.dm

```diff
@@ -62,6 +62,16 @@ one type of shotgun ammo, but I think it helps in referencing it. ~N
 	w_class = WEIGHT_CLASS_SMALL // CAN throw it in your pocket, friend.
 	icon_state_mini = "mosin"
 
+/obj/item/ammo_magazine/rifle/boltclip //Nearly 1:1 copy of above
+	name = "clip of 7.62x54mmR rifle rounds"
+	desc = "A Disposible Stripper clip filled with rifle bullets."
+	icon_state = "clip"
+	default_ammo = /datum/ammo/bullet/sniper/svd
```

Comment:
who tf made mosin use svd ammo suffering

Reply:
I assume it was the person who originally made the gun itself in the code cause that is what the gun is listed as taking.  Im just trying to make bullet holders, not change the ammo type.